### PR TITLE
Disable the ability to add bold to headings

### PR DESCRIPTION
**Pull request by MadhavKrishanGoswami**
# ✨ Codu Pull Request 💻

Codu Logo

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## Pull Request details:
Disabled the ability to add bold to headings. No bold/italic/strikethrough tags should be present in headers
## Any Breaking changes:
None

## Associated Screenshots:
None

Fixed#566

### CODE REVIEW

**Comment by pkspyder007** [comment]
@MadhavKrishanGoswami  Can you please explain what does your change affects?
Screenshots/recording are really helpful.

**Comment by NiallJoeMaher** [comment]
Will test this later today. 🦾

File: hooks/useCreatePage.ts [hunk 1]
```diff
@@ -202,6 +202,10 @@ function useCreatePage({
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     e.preventDefault();
     const { value } = e.target;
+    const isTitleElement = e.target.dataset.type === "title";
```
Comment (NiallJoeMaher) [inline hooks/useCreatePage.ts:205]: @xiaoniuniu89 do you think this will work or break anything that I'm not seeing?

Comment (xiaoniuniu89) [reply hooks/useCreatePage.ts:205]: this onChange handler looks as if it is for creating tags? the onChange for the editor is handled by react hook form. 
Are you sure this is the behaviour you want? I think I am missing something

Comment (xiaoniuniu89) [reply hooks/useCreatePage.ts:205]: I am thinking you can handle this from the TipTapExstenstions file **or** from the Editor.tsx before calling onChange

Comment (NiallJoeMaher) [reply hooks/useCreatePage.ts:205]: That's what I assumed too! @MadhavKrishanGoswami would you be able to check if this works as expected and add screenshots?

**Comment by NiallJoeMaher** [comment]
Closing since there was no reply to this one in a while.

**Pull request closed without merging**